docs/en-US/component/alert.md

```
@@ -79,26 +79,22 @@ alert/icon-description
-| closable                           | whether alert can be dismissed.          | ^[boolean]                                                                  | true    |
-| center                             | whether content is placed in the center. | ^[boolean]                                                                  | false   |
-| close-text                         | customized close button text.            | ^[string]                                                                   | —       |
-| show-icon                          | whether a type icon is displayed.        | ^[boolean]                                                                  | false   |
-| effect                             | theme style.                             | ^[enum]`'light' \| 'dark'`                                                  | light   |
-| show-after ^(2.10.0) ^(deprecated) | delay of appearance, in millisecond      | ^[number]                                                                   | 0       |
-| hide-after ^(2.10.0) ^(deprecated) | delay of disappear, in millisecond       | ^[number]                                                                   | 200     |
-| auto-close ^(2.10.0) ^(deprecated) | timeout in milliseconds to hide alert    | ^[number]                                                                   | 0       |
+| Name        | Description                              | Type                                                                        | Default |
+| ----------- | ---------------------------------------- | --------------------------------------------------------------------------- | ------- |
+| title       | alert title.                             | ^[string]                                                                   | —       |
+| type        | alert type.                              | ^[enum]`'primary' (2.9.11) \| 'success' \| 'warning' \| 'info' \| 'error' ` | info    |
```

markdownlint-cli2 (0.18.1)

Spaces inside code span elements

(MD038, no-space-in-code)